apps/ai-content-generator/src/configs/features/featureTypes.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import { AIFeature } from './featureConfig';
+
+/**
+ * Core Types
+ */
+
+export type Prompt = (input: string, targetLocale: string) => string;
```

Comment:
Should this be exported? It's only be used in this file right now

Reply:
These exports will be used in the following PRs